api/src/main/java/io/druid/data/input/impl/ParseSpec.java

```diff
@@ -22,12 +22,12 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonSubTypes;
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
-
 import io.druid.java.util.common.parsers.Parser;
 
 import java.util.List;
 
 /**
+ * Not an {@code ExtensionPoint} since extension parsers are meant to extend {@code InputRowParser}.
```

Comment:
That's a good point…

I guess ParseSpec should be an extension point? And that would mean TimestampSpec and DImensionsSpec should be public apis.

I'll change it.